aula-pratica-02/ex2.c

```diff
@@ -0,0 +1,28 @@
+#include<stdio.h>
+#include<stdlib.h>
+int primo(int n);
+
+int primo(int n)
+{
+    for (int i = 2; i < n; i ++) {
+        if (n % i == 0) {
+            return 0;
+        }
+    }
+    return 1;
+}
+
+int main()
+{
+    int n, soma = 0, cont = 0;
+    scanf("%d", &n);
+    for(int i = 2; cont < n; i++){
+        if(primo(i) == 1){
+            soma += i;
+            cont++;
+        }
+    }
+    printf("Soma dos %d primeiro n�meros primos: %d", n, soma);

```

Comment:
```suggestion
    printf("Soma dos %d primeiro números primos: %d\n", n, soma);
```

it's always a good thing end the sentences with a line break, otherwise the output of your program on the command line gets messed up